core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/Server.json

```diff
@@ -235,7 +235,7 @@
             "attributes": {
                 "AdditionalMigrationAttempts":  [ {"version": "[10,)",     "wlst_mode": "both",    "wlst_name": "AdditionalMigrationAttempts",  "wlst_path": "WP001", "default_value": 2,        "wlst_type": "integer"  } ],
                 "AllCandidateServers":          [ {"version": "[10,)",     "wlst_mode": "online",  "wlst_name": "AllCandidateServers",          "wlst_path": "WP001", "default_value": null,     "wlst_type": "jarray",           "access": "IGNORED", "preferred_model_type": "list", "get_method": "GET", "set_method": "MBEAN.set_server_mbeans", "set_mbean_type": "weblogic.management.configuration.ServerMBean"} ],
-                "Cluster":                      [ {"version": "[10,)",     "wlst_mode": "both",    "wlst_name": "Cluster",                      "wlst_path": "WP001", "default_value": null,     "wlst_type": "string",           "get_method": "${LSA:GET}", "set_method": "MBEAN.set_cluster_mbean", "set_mbean_type": "weblogic.management.configuration.ClusterMBean"} ],
+                "Cluster":                      [ {"version": "[10,)",     "wlst_mode": "both",    "wlst_name": "Cluster",                      "wlst_path": "WP001", "default_value": null,     "wlst_type": "string",           "get_method": "${LSA:GET}", "set_method": "${MBEAN.set_server_cluster_mbean:MBEAN.set_cluster_mbean", "set_mbean_type": "weblogic.management.configuration.ClusterMBean"} ],
```

Comment:
needs closing }

Reply:
Are we supposed to change Server.json or ServerTemplate.json ?